src/main/java/io/cdap/e2e/utils/ElementHelper.java

```diff
@@ -73,7 +73,7 @@ public static void clickOnElementUsingJsExecutor(WebElement element) {
    */
   public static void clickOnElement(WebElement element) {
     WaitHelper.waitForElementToBeClickable(element);
-    scrollToElement(element);
+    scrollToElementUsingJsExecutor(element);
```

Comment:
Why is this change needed? 
Using js executor does not simulate user actions.

Reply:
Normal `scrollToElement(element)` is not working in new left side panel.

<img width="274" alt="image" src="https://user-images.githubusercontent.com/94541628/185431023-7e40a480-f294-46ca-980e-136f6268ee1d.png">

Reply:
This change might be related to the recent migration of Pipeline Studio from angular to react.
But we cannot make this action use JS Executor since this is being used across all tests.
Please check with @mrahanjam & @sumengwang.

Reply:
We want to fix the ui to match the e2e tests as much as possible. It will be good to just identify the issues first

Reply:
Can we please open a JIRA to track this? I am fine with this change if it is temporary but in longer term using JS Executor might not be the correct way to interact with UI elements in end-to-end tests.

Reply:
https://github.com/cdapio/cdap-ui/pull/605/files
this should fix the scroll issue. The footer will no longer block the elements. no need for jsexecutor

Reply:
Removed use of JS Executor. Above change to fix scroll issue is not available in latest sandbox successful build. @itsankit-google Please re-trigger e2e tests after next successful sandbox build.